documentation/modules/uninstalling-mtv-ui.adoc

```diff
@@ -6,18 +6,59 @@
 [id="uninstalling-mtv-ui_{context}"]
 = Uninstalling {project-short} by using the {ocp} web console
 
-You can uninstall {project-first} by using the {ocp} web console to delete the +{namespace}+ project and custom resource definitions (CRDs).
+You can uninstall {project-first} by using the {ocp} web console.
 
 .Prerequisites
 
 * You must be logged in as a user with `cluster-admin` privileges.
 
 .Procedure
 
-. Click *Home* -> *Projects*.
-. Locate the *{namespace}* project.
-. On the right side of the project, select *Delete Project* from the {kebab}.
-. In the *Delete Project* pane, enter the project name and click *Delete*.
-. Click *Administration* -> *CustomResourceDefinitions*.
-. Enter `forklift` in the *Search* field to locate the CRDs in the `forklift.konveyor.io` group.
-. On the right side of each CRD, select *Delete CustomResourceDefinition* from the {kebab}.
+. In the {ocp} web console, click *Operators > Installed Operators*.
+. Click *{project-full} Operator*.
++
+The *Operator Details* page opens in the *Details* tab.
+. Click the *ForkliftController* tab.
+. Click *forklift-controller*.
++
+The *ForkliftController details* page opens.
```

Comment:
```suggestion
The *ForkliftController details* page opens.
```
Since I have not seen the UI, can you please confirm if the name of the page is *ForkliftController details*? I am just being sure.

Reply:
Good call.